pkg/dashboard/functiontemplates/types.go

```diff
@@ -35,3 +35,14 @@ type generatedFunctionTemplate struct {
 	SourceCode         string
 	serializedTemplate []byte
 }
+
+// keeps the file contents of a function template
+// can keep 2 types of template formats:
+// 1. Code and TemplateAndValues
+// 2. Code, Template and Values
+type FunctionTemplateFileContents struct {
+	Code              string
+	Template          string
+	Values            string
+	TemplateAndValues string
```

Comment:
i think we should do the extra work to remove `TemplateAndValues` from the struct (meaning changing `generated` and aligning all templates to one format)